### PR TITLE
Validate manual schema

### PR DESCRIPTION
**Pull request by itchyny**
There has been some mistakes in manual files so far (`example` vs `examples`, or plain value in `output`). We should automate the schema validation to reduce frustration of time wasting review communication.

### CODE REVIEW

**Comment by nicowilliams** [comment]
This is very nice!  Thank you!  Should this be merged after #2744?

File: Makefile.am [hunk 1]
```diff
@@ -152,7 +152,9 @@ check_DATA = tests/man.test
 # don't care. But if you are, then you need to run the tests anyway.
 tests/man.test tests/manonig.test: $(srcdir)/docs/content/manual/manual.yml
 if ENABLE_DOCS
-	$(AM_V_GEN) ( cd ${abs_srcdir}/docs; $(PIPENV) run python build_mantests.py )
+	$(AM_V_GEN) ( cd ${abs_srcdir}/docs; \
+		$(PIPENV) run python validate_manual_schema.py content/manual/manual.yml; \
+		$(PIPENV) run python build_mantests.py )
```
Comment (wader) [inline Makefile.am:157]: Use .. && … to fail on validate error? Maybe already does somehow?

Comment (itchyny) [reply Makefile.am:157]: Yeah, should be `&&`. Thanks.

**Comment by itchyny** [comment]
> Should this be merged after https://github.com/jqlang/jq/pull/2744?

I tested locally see both PRs doesn't conflict and merge order doesn't matter (merging both into a single branch also passes the validation).

**Comment by nicowilliams** [comment]
> I tested locally see both PRs doesn't conflict and merge order doesn't matter (merging both into a single branch also passes the validation).

Thanks!

**Pull request merged**